### DISCUSSION TITLE
Naming theme on install, still see some tailpress references

### DISCUSSION THREAD

**Discussion by ineptian** [tailpress/tailpress#230] [Q&A]
I was just wondering if this is an issue or not. When naming the theme during composer install, I still see some references to tailpress in some places.

For example on content, content-single, the post navigation. Should that be updated to the new text domain?
```php
wp_link_pages(
	array(
		'before' => '<div class="page-links"><span class="page-links-title">' . __( 'Pages:', 'tailpress' ) . '</span>',
		'after' => '</div>',
		'link_before' => '<span>',
		'link_after' => '</span>',
		'pagelink' => '<span class="screen-reader-text">' . __( 'Page', 'tailpress' ) . ' </span>%',
		'separator' => '<span class="screen-reader-text">, </span>',
	)
);
```

**Comment by jeffreyvr** [comment]
Should be replaced ideally indeed. As a reference to myself - or is someone wants to contribute - this should be added here:

https://github.com/jeffreyvr/tailpress-installer/blob/master/src/Console/NewCommand.php#L85
